repo/packages/M/mongodb-replicaset/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I know, it's currently listed as optional (deprecated) in the Universe docs. I added it because it's in the dcos-commons Universe template files...

The latest Kafka framework also contains a [command.json](https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kafka/35/command.json) and has `packagingVersion` set to [3.0](https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kafka/35/package.json#L7). It's be great if this could be consistent... My suggestion would be to either update the docs regarding this, or allow the command.json in `3.0`